src/appComponents/common/ApiJsonFactory.js

```diff
@@ -15,31 +15,30 @@ var baseUrl = (config.use_testApi ? "testapi/" : config.apiBaseGetUrl) + "event/
                 let endpoint = "";
 
               if ($apiEndpoint === "event" && !config.use_testApi) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["config" is not defined.](https://www.codacy.com/app/triplez/open-event-webapp/file/2444008392/issues/source?bid=3233857&fileBranchId=3245777#l17)